app/configurator/components/chart-type-selector.tsx

```diff
@@ -101,6 +103,10 @@ const ChartTypeSelectorField = ({
       disabled={disabled}
       label={label}
       {...field}
+      onClick={(e) => {
+        setChartError("none");
+        field.onClick(e);
+      }}
```

Comment:
I guess it's temporary, I will think about the way to handle that properly. This probably should happen inside configuratorState, on "CHART_TYPE_CHANGED" action type, but here I can't use a useChartError hook directly in reducer, so I think a bigger refactor would be needed?